experimenter/experimenter/targeting/constants.py

```diff
@@ -2342,9 +2342,19 @@ def __post_init__(self):
     description="Target users who have never used the new or old sidebar",
     targeting=(
         "!('sidebar.revamp'|preferenceValue) && "
-        "'sidebar.backupState'|preferenceValue == '{}' &&"
+        "!('browser.engagement.sidebar-button.has-used'|preferenceValue) && "
         "primaryResolution.width > 1366 &&"
-        "primaryResolution.height > 768"
+        "primaryResolution.height > 768 &&"
```

Comment:
With this implicit concatenation, presumably each line needs a space before the closing quote on each line except the last one to avoid a result like `&&addonsInfo.addons` etc?

Reply:
Ah, good catch!